Test/Tasty/Silver/Interactive.hs

```diff
@@ -80,23 +83,28 @@ type DisabledTests = TestPath -> Bool
 
 -- | Like @defaultMain@ from the main tasty package, but also includes the
 -- golden test management capabilities.
+
 defaultMain :: TestTree -> IO ()
```

Comment:
If I am not mistaken, if the terminal is not interactive all golden values are updated but the exit code is 0 (aka success). If that is indeed the case, maybe you can mention this in the haddock comment here. E.g. something like this:

> ....
> If a golden test fails, the user can interactively analyse the difference and optionally choose to update the golden value.
>
> NOTE: It is not recommended to use the interactive mode in batch processes like e.g. CI builds, as it may hide failing golden tests.

Reply:
Reverted this change.